cl/settings/project/logging.py

```diff
@@ -78,6 +78,16 @@ def skip_unreadable_post(record):
         },
         # This is the one that's used practically everywhere in the code.
         "cl": {"handlers": ["console"], "level": "INFO", "propagate": True},
+        "juriscraper": {
+            "handlers": ["console"],
+            "propagate": True,
+            "level": "DEBUG",
+        },
+        "Logger": {
+            "handlers": ["console"],
+            "propagate": True,
+            "level": "DEBUG",
+        },
```

Comment:
I'm pretty bad at loggers, but I think my understanding is that by putting the `juriscraper` logger here, CL will log out juriscraper logs? 

What's the `Logger` logger for? 

Do we need DEBUG for both of these? That feels like it could (and should) be noisy. Better to stick with INFO?

Reply:
The `Logger` is a named logger from `make_default_logger` in `juriscraper/lib/log_utils.py`. It is the only logger used in all of juriscraper, so we can just delete the "juriscraper" logger entry.
https://github.com/freelawproject/juriscraper/blob/aff4c45150d9649871aa9aed5889fd3c074ba776/juriscraper/lib/log_tools.py#L16-L23

[Usually](https://docs.python.org/3/library/logging.html#logger-objects) loggers are defined using `logger = logging.getLogger(__name__)`  which gives it a name with the full module path, and allows for granularity in selecting what we log, but we don't do that in juriscraper

About it logging everything, since the only handler defined for it is the "console" handler, defined as a          `class": "logging.StreamHandler"`, I expect it only to appear in the console where the command is being executed, which is the use case we want to cover, to have all the logs when we are debugging by running something manually. Via the same reasoning, we set the log level to "DEBUG"

Reply:
Wouldn't console logging also add this to all of our server logs? I think they just log to the console too?